scaladoc/src/dotty/tools/scaladoc/renderers/Renderer.scala

```diff
@@ -82,34 +74,17 @@ abstract class Renderer(rootPackage: Member, val members: Map[DRI, Member], prot
             )
           updatedTemplates.result()
 
-        val newTemplates = updateSettings(templates, newSettings.to(ListBuffer))
+        val newTemplates = updateSettings(Seq(siteContext.staticSiteRoot.rootTemplate), newSettings.to(ListBuffer))
```

Comment:
I guess it was supposed to be `Seq(rootTemplate)` or wh do we need property in line 43?